src/DssSpell.sol

```diff
@@ -47,13 +51,69 @@ contract DssSpellAction is DssAction {
     uint256 constant MILLION = 10**6;
 
     function actions() public override {
-        //                           ilk            line           gap       ttl
-        setIlkAutoLineParameters("ETH-A", 1500 * MILLION, 30 * MILLION, 12 hours);
+        // add UNI-V2-DAI-USDC-A collateral type
+        CollateralOpts memory UNIV2DAIUSDC_A = CollateralOpts({
+            ilk: "UNIV2DAIUSDC-A",
+            gem: 0xAE461cA67B15dc8dc81CE7615e0320dA1A9aB8D5,
+            join: 0xA81598667AC561986b70ae11bBE2dd5348ed4327,
+            flip: 0x4a613f79a250D522DdB53904D87b8f442EA94496,
+            pip: 0x25CD858a00146961611b18441353603191f110A0,
+            isLiquidatable: false,
+            isOSM: true,
+            whitelistOSM: false,
+            ilkDebtCeiling: 3 * MILLION, // initially 3 million
+            minVaultAmount: 2000,
+            maxLiquidationAmount: 50000,
+            liquidationPenalty: 1300,
+            ilkStabilityFee: 1000000000937303470807876289, // 3%
+            bidIncrease: 300, // 3%
+            bidDuration: 6 hours,
+            auctionDuration: 6 hours,
+            liquidationRatio: 11000 // 110%
+        });
+        addNewCollateral(UNIV2DAIUSDC_A);
+
+        addReaderToMedianWhitelist(
```

Comment:
In the future, @NiklasKunkel can probably do this for LP tokens prior to denying himself.

Reply:
nvm, he doesn't have medianizer access. We'll just have to remember this for LP onboarding. Our tests catch it if we don't.